### DISCUSSION TITLE
Saving communication log

### DISCUSSION THREAD

**Discussion by iliavls** [jas3333/GPT-Helper#25] [Ideas]
Hello! Sorry to bother, but would be really great to implement the possibility to save the dialog between bot and user. I asked GPT itself and he said it's possible 😂 Also he promised he would not forget about task I gave him if I close the window and he actually forgot. Would be also great if you could put somewhere inside of app or Github page the description about different setting we have (temperature, threads, drop-down menu etc). Your app is amazing and thank you for sharing it!

**Comment by jas3333** [comment]
I just pushed an update that uses localStorage to save the conversation. You can use the reset button to remove it. You can hover over the settings and a tooltip should pop up explaining what each setting does. The one setting that confuses me because I haven't spent a whole lot of time with it lately is the token amount. 

Hopefully this helps you out, glad you're enjoying it. Sorry for the late response I've been distracted with trying to learn devops.
